Content.Server/Doors/Components/AirlockComponent.cs

```diff
@@ -158,16 +158,35 @@ public void UpdateWiresStatus()
                 return;
             }
 
-            var powerLight = new StatusLightData(Color.Yellow, StatusLightState.On, "POWR");
-            if (PowerWiresPulsed)
+            StatusLightData powerLight;
+            if(WiresComponent != null)
             {
-                powerLight = new StatusLightData(Color.Yellow, StatusLightState.BlinkingFast, "POWR");
+                var mainPowerCut = WiresComponent.IsWireCut(Wires.MainPower);
+                var backupPowerCut = WiresComponent.IsWireCut(Wires.BackupPower);
+                var statusLightState = PowerWiresPulsed ? StatusLightState.BlinkingFast : StatusLightState.On;
+                if (mainPowerCut && backupPowerCut)
+                {
+                    powerLight = new StatusLightData(Color.Yellow, StatusLightState.Off, "POWR");
+                }
+                else if(mainPowerCut != backupPowerCut)
+                {
+                    powerLight = new StatusLightData(Color.Yellow, statusLightState, "POWR");
+                }
+                else
+                {
+                    powerLight = new StatusLightData(Color.Red, statusLightState, "POWR");
```

Comment:
So previously it was:
pulsed: blinking yellow
none cut: solid yellow
one cut: still solid yellow
both cut: solid red

now its:
pulsed: blinking red
none cut: solid red
one cut: solid yellow
both cut: off

Just to keep things consistent with how they were before, wouldn't it make more sense to use:
pulsed: blinking yellow
none cut: solid yellow
one cut: solid red
both cut: off

At the very least, if you want to use red as the new "default", maybe consider changing either safety or bolt to another colour, as otherwise 3/6 of the lights are all (initially) using red.

Reply:
Changed the colors